packages/contracts/src/dollar/facets/DollarAmoMinterFacet.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: MIT
```

Comment:
PR it's missing natspec in all implementations

Reply:
fixed